.travis.yml

```diff
@@ -1,6 +1,6 @@
 ---
 language: "node_js"
-node_js: [ "10", "11", "12", "13", "node" ]
```

Comment:
why would you want to drop support for older node versions? testing on multiples happens in parallel, so it's not even slowing anything down (since you're at the parallelization threshold of 5)

`12` is already the latest LTS, so it's already tested.

Reply:
Yeah, this should be an addition rather than a replacement.

Reply:
Since 14 isn't LTS yet, what you've got now is `10, 11, 12, 13, 12`. In other words, I don't think any change is needed here; you're already testing everything above 10.

`lts/*` isn't a range, it *selects* the latest LTS version (which is 12)

Reply:
Understood and intended. I want to catch LTS in the future without bumping `.travis.yml` all the time.

I suppose I should replace `node`, as well.

Reply:
makes sense. you also may want to add an explicit "engines" field - `npx ls-engines` :-)